README.md

```diff
@@ -69,15 +70,15 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * Updated [plugin authors](https://gist.github.com/glueckpress/f058c0ab973d45a72720)
 
 ### 2.2.2 ###
-* Fix: Parameter-Rückgabe beim Filter `dashboard_glance_items`
-* Großzügige Anwendung des Filters `esc_html`
+* Fix: parameter response by filter `dashboard_glance_items`
+* Generous use of the filter `esc_html`
 
 ### 2.2.1 ###
-* Fix für die Meldung "Call to undefined function is_plugin_active_for_network" in WordPress-Multisite
+* Fix for the note "Call to undefined function is_plugin_active_for_network" on WordPress-Multisites
```

Comment:
I assume it was originally a PHP _notice_, not _note_.

Also: _WordPress Multisite_, not _WordPress-Multisites_